bindings/python/src/session.cpp

```diff
@@ -57,61 +57,61 @@ namespace lt = libtorrent;
 namespace
 {
 #ifndef TORRENT_NO_DEPRECATE
-    void listen_on(lt::session& s, int min_, int max_, char const* interface, int flags)
```

Comment:
would you mind reverting the white-space change? It dilutes the actual diff. At some point the spaces in here should be turned into tabs, but that should be a white-space-only patch